### PR TITLE
kubernetes: Move nodes out of details

### PR DESCRIPTION
**Pull request by petervo**

### CODE REVIEW

File: pkg/kubernetes/scripts/nodes.js [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function() {
+    "use strict";
+
+    var VOLUME_FACTORY_SUFFIX = "VolumeFields";
```
Comment (stefwalter) [inline pkg/kubernetes/scripts/nodes.js]: I don't think this is necessary.

File: pkg/kubernetes/views/node-page.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div class="content-filter">
+    <h3 ng-if="item"><i class="fa fa-database fa-fw"></i>{{ item.metadata.name }}</h3>
```
Comment (stefwalter) [inline pkg/kubernetes/views/node-page.html]: This should be ```pficon pficon-container-node```

File: pkg/kubernetes/views/node-page.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <h3 ng-if="!item"><i class="fa fa-database fa-fw"></i>{{ target }}</h3>
```
Comment (stefwalter) [inline pkg/kubernetes/views/node-page.html]: Ditto.

**Comment by stefwalter** [comment]
Needs a rebase.

Should this have its own link on the sidebar?

**Comment by andreasn** [comment]
> Should this have its own link on the sidebar?

Yeah, I think that would make sense.

**Comment by petervo** [comment]
Made changes and added to the side bar, i was trying too keep it out till after the redesign because there is so much there but see what you think.

**Pull request closed without merging**